.github/workflows/pytorch-linux-bionic-rocm4.2-py3.6.yml

```diff
@@ -0,0 +1,403 @@
+# @generated DO NOT EDIT MANUALLY
+# Template is at:    .github/templates/linux_ci_workflow.yml.j2
+# Generation script: .github/scripts/generate_ci_workflows.py
+name: Linux CI (pytorch-linux-bionic-rocm4.2-py3.6)
+
+on:
+  # TODO: Enable pull_request builds when we can verify capacity can be met by auto-scalers
+  push:
+    branches:
+      - master
+      - release/*
+  workflow_dispatch:
+
+env:
+  BUILD_ENVIRONMENT: pytorch-linux-bionic-rocm4.2-py3.6
+  DOCKER_IMAGE_BASE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-bionic-rocm4.2-py3.6
+  SCCACHE_BUCKET: ossci-compiler-cache-circleci-v2
+  TORCH_CUDA_ARCH_LIST: 5.2
```

Comment:
@seemethere This env variable is not relevant for ROCm CI. However, should the equivalent `PYTORCH_ROCM_ARCH` variable be set to `gfx900;gfx906` here based on https://github.com/pytorch/pytorch/blob/master/.jenkins/pytorch/build.sh#L151?